src/Tag/index.scss

```diff
@@ -0,0 +1,30 @@
+.nds-tag {
+  padding:var(--space-xs);
+  border-radius: 16px;
+  display: flex;
+  align-items: baseline;
+  font-size: var(--font-size-xs);
+  .narmi-icon-x {
+    color: var(--color-white);
+    cursor: pointer;
+    margin-left: 8px;
```

Comment:
This needs to be margin instead of padding because the padding space makes the clickable area bigger

Reply:
good catch!